### ISSUE TITLE
Invited users are unable to register

### ISSUE THREAD

**Issue by bouvet-stein** [getsentry/self-hosted#1173]
### Version

21.11.0

### Steps to Reproduce

1. Install Sentry on.premise on Azure VM
2. Log in as superuser
3. Go to Settings -> Members, and invite a user to the Sentry installation
4. The user receives the invite email, and attempts to register an account. Filling in name, email address and password, and clicks the Register button
5. The user waits and waits, the Register page hangs

I have also tried to reset the entire installation using the reset.sh script, but the behavior is the same after the reset. Also tried to set the invited member as Admin (as compared to the normal Member), but this had no effect.

### Expected Result

The invited user is able to register an account

### Actual Result

The user waits and waits, the Register page hangs. So the user is unable to accept the invite and register an account.

![image (7)](https://user-images.githubusercontent.com/89916958/143508771-4cee86a9-b030-404a-a274-c5806d108e38.png)

**Comment by chadwhitacre** [comment]
> the Register page hangs

Any error showing up in the browser console and/or network tab?

**Comment by bouvet-stein** [comment]
No. And this issue was resolved on Friday, and the reason was that Sentry was places behind a firewall on Azure which used IDPS and Alert&Deny enabled. So the firewall categorized some of the traffic to Sentry as dangerous and blocked it. The entries in the firewall log was like this:
"drop. Signature: 2101201. IDS: WEB_SERVER 403 Forbidden. Priority: 2. Classification: Attempted Information Leak"

So the Azure firewall was a bit too smart, we had to put in a bypass rule for all traffic to Sentry. Then everything worked as it should, including user registration.

**Comment by chadwhitacre** [comment]
Nice, glad you got it sorted ... and thanks for posting back for others to find in the future! :)
